Gemfile.jruby-2.6.lock.release

```diff
@@ -94,7 +94,6 @@ GEM
     concurrent-ruby (1.1.9)
     crack (0.4.5)
       rexml
-    csv (3.2.5)
```

Comment:
@mashhur [8.4](https://github.com/elastic/logstash/blob/8.4/versions.yml#L16) branch has updated to jruby 9.3.8.0 to fix csv thread leak issue. I think we can remove the pin version.